provider/testdata/data/cldr/cldr-core/supplemental/metaZones.json

```diff
@@ -2,7 +2,7 @@
   "supplemental": {
     "version": {
       "_unicodeVersion": "14.0.0",
-      "_cldrVersion": "41"
+      "_cldrVersion": "40"
```

Comment:
This was included manually in #1782, I've added it to the CLDR_JSON_GLOB and reran the download.